common/src/unifycr_const.h

```diff
@@ -105,14 +105,16 @@
 // #define IP_STR_LEN 20
 // #define MAX_CQ_SIZE 2*NUM_CLI
 // #define MAX_NUM_CONNS 10
-// #define NUM_MSG 32768 /* number of messages one server sends to each of another server */
+// /* number of messages one server sends to each of another server */
+// #define NUM_MSG 32768
 // #define NUM_OF_READ_TASKS 65536
 // #define PORT_STR_LEN 10
 // #define SH_BUF_SIZE 1048576
-// #define SERVICE_MEM_POOL_SIZE 2147483648 /* buffer size on the service manager for read clustering and pipelining */
+// /* buffer size on the service manager for read clustering and pipelining */
+// #define SERVICE_MEM_POOL_SIZE 2147483648
 
 /* ****************** Metadata/MDHIM Default Values ******************** */
-#define META_DEFAULT_DB_PATH /tmp
+#define META_DEFAULT_DB_PATH (/tmp)
```

Comment:
This jumped out at me. Making sure it's intentional.

Reply:
Yeah, that was forced by checkpatch.

Reply:
https://docs.freebsd.org/info/cpp/cpp.info.Macro_Parentheses.html

Reply:
Well, I should have tested that better.  Adding the parens leads to a runtime error.  Trying quotes instead.

Reply:
Well quotes need some help, too...
```
+ /g/g0/moody20/projects/unifycr2.git/install/bin/unifycrd
# accepting RPCs on address "na+sm://146417/0"
completed sock init server
MLOG CRIT Error opening leveldb database, abc..., path is "/tmp"/unifycr_db-0-0
MLOG CRIT MDHIM Rank: 0 - Error while opening database
MLOG CRIT Rank: 0 - Error opening data store for index: 0
MLOG CRIT Error opening leveldb database, abc..., path is "/tmp"/unifycr_db-1-0
MLOG CRIT MDHIM Rank: 0 - Error while opening database
MLOG CRIT Rank: 0 - Error opening data store for index: 1
```

Reply:
Looking at that link @dsikich posted, I wonder if checkpatch thought the "/" was a mathematical operation and not a path.

Reply:
This particular MACRO that defines the DB path is only used in common/src/unifycr_configurator.h.  We could likely avoid this problem by hardcoding the string in the UNIFYCR_CFG statement in that file.  I'll test that.

Reply:
Makes me wonder what Travis CI is doing differently (or wrong) that allowed it to run unifycrd.